frontend/src/components/cronjob/Details.tsx

```diff
@@ -12,6 +19,22 @@ export default function CronJobDetails() {
   const { t, i18n } = useTranslation('glossary');
 
   const [jobs, jobsError] = Job.useList();
+  const [cronJob, cronJobError] = CronJob.useGet(name, namespace);
```

Comment:
We should avoid doing this because it'll fetch the details twice from the server: once here, and another time as part of the DetailsGrid. I think that since you need to do all these changes, probably you should replace the DetailsGrid by MainInfoSection (which does less things automatically but it's still easy to use).

Reply:
updated with MainInfoSection